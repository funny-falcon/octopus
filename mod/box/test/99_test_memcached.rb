#!/usr/bin/ruby1.9.1

$:.push 'test/lib'
require 'standalone_env'

class Env < StandAloneEnv
  def config
    <<EOD
pid_file = "octopus.pid"

slab_alloc_arena = 1
logger = "exec cat - >> octopus.log"
log_level = 5
primary_port = 33013
wal_fsync_delay = 10

memcached = 1
object_space = []

EOD
  end
end

Env.clean.with_server do
  ENV["T_MEMD_USE_DAEMON"]="127.0.0.1:33013"
  result = `prove ../../../../third_party/memcached/t`
  puts result.gsub(/, \d+ wallclock.*/, "")
end
