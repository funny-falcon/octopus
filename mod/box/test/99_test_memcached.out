../../../../third_party/memcached/t/binary-get.t ....... ok
../../../../third_party/memcached/t/bogus-commands.t ... ok
../../../../third_party/memcached/t/cas.t .............. ok
../../../../third_party/memcached/t/expirations.t ...... ok
../../../../third_party/memcached/t/flags.t ............ ok
../../../../third_party/memcached/t/flush-all.t ........ ok
../../../../third_party/memcached/t/getset.t ........... ok
../../../../third_party/memcached/t/incrdecr.t ......... ok
../../../../third_party/memcached/t/multiversioning.t .. ok
../../../../third_party/memcached/t/noreply.t .......... ok
All tests successful.
Files=10, Tests=665
Result: PASS
